control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -194,12 +278,21 @@ class Pid
    * \param i_min Lower integral clamp.
    *
    */
-    [[deprecated("Use constructor with AntiwindupStrategy instead.")]]
+    [[deprecated("Use constructor with AntiWindupStrategy instead.")]]
     Gains(double p, double i, double d, double i_max, double i_min)
-    : Gains(
-        p, i, d, i_max, i_min, std::numeric_limits<double>::infinity(),
-        -std::numeric_limits<double>::infinity(), 0.0, true, AntiwindupStrategy::NONE)
+    : p_gain_(p),
+      i_gain_(i),
+      d_gain_(d),
+      i_max_(i_max),
+      i_min_(i_min),
+      u_max_(std::numeric_limits<double>::infinity()),
+      u_min_(-std::numeric_limits<double>::infinity()),
+      antiwindup_(false)
     {
+      antiwindup_strat_.type = AntiWindupStrategy::LEGACY;
+      antiwindup_strat_.i_max = i_max;
+      antiwindup_strat_.i_min = i_min;
+      antiwindup_strat_.legacy_antiwindup = false;
```

Comment:
```suggestion
      antiwindup_strat_.legacy_antiwindup = true;
```
wasn't true here the default?